packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -503,8 +506,8 @@ The DomainResponse object defines the following properties:
 
 | Property                  | Type                                                           | Description                                                                               |
 | ------------------------- | -------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
-| `certificateSourceType`   | `MANUAL`                                                       | Certificate source type that indicates whether the Certificate is provided by the user. Accepted value: `MANUAL`|
-| `dnsRecords`              | Array of [DNSRecord](#dnsrecord-object)                        | TXT and CNAME records to be registered for the Domain                                     |
+| `certificateSourceType`   | String                                                       | Certificate source type that indicates whether the certificate is provided by the user or Okta.                                     |
```

Comment:
no accepted values that need to be listed here? Accept values: `MANUAL`, `OKTA_MANAGED`??

Reply:
This is a response object